### PR TITLE
fix(sampling): Fix flaky e2e test - [TET-273]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Update sampling E2E test 

Final solution: add `click_when_visible` so we can wait element before clicking on it.

https://github.com/getsentry/sentry/blob/0e8ec4d6c1114199dff054af9a2021ff122c1bae/src/sentry/utils/pytest/selenium.py#L176-L186

### CODE REVIEW

**Review by matejminar — APPROVED** [review]
🤞

**Review by ahmedetefy — APPROVED** [review]

File: tests/acceptance/test_project_settings_sampling.py [hunk 1]
```diff
@@ -103,27 +103,30 @@ def test_add_uniform_rule_with_recommended_sampling_values(self):
...
-            self.browser.element('[aria-label="Start Setup"]').click()
+            self.browser.click_when_visible('[aria-label="Start Setup"]')
+
+            self.browser.wait_until('[id="recommended-client-sampling"]')
 
             # Enter a custom value for client side sampling
-            self.browser.element('[id="recommended-client-sampling"]').clear()
-            self.browser.element('[id="recommended-client-sampling"]').send_keys(80, Keys.ENTER)
+            self.browser.element('[id="recommended-client-sampling"]').send_keys(
+                Keys.BACK_SPACE, Keys.BACK_SPACE, Keys.BACK_SPACE, 80, Keys.ENTER
```
Comment (ahmedetefy) [inline tests/acceptance/test_project_settings_sampling.py:117]: This does not seem like good practice to me, why are you calling back space three times here?

Comment (andriisoldatenko) [reply tests/acceptance/test_project_settings_sampling.py:117]: yeah, i can remove it back to original version.

Comment (ahmedetefy) [reply tests/acceptance/test_project_settings_sampling.py:117]: please do then, and this one too 7ff3361ad86b28f71d6915a47fe070a633feec806854b92a90d0fd0d3a8e2403R122 ?

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**